### DISCUSSION TITLE
HTTP/3 CDN

### DISCUSSION THREAD

**Discussion by trusktr** [jspm/project#295] [Ideas]
What's the plan for this?

I'm really curious if this may improve multiplexing minified/gzipped ES modules over a single stream (in ways HTTP/2 could not).

So far Node, Deno, and Bun do not have HTTP/3 yet, so people haven't really tried this I think (and the people who would want to try this the most probably don't want to switch from JS to another language like Rust, although it isn't out of the realm of possibility).

**Comment by guybedford** [comment]
It should definitely be an improvement for parallelism in these models, I don't think there's any doubt about that. It's more of an infrastructure question when we can move to an HTTP/3 CDN, either through our existing CDN sponsor, or through new infrastructure directions in future.
